circuits/credentialAtomicQuerySigOnChainSmt.circom

```diff
@@ -0,0 +1,15 @@
+pragma circom 2.0.0;
+
+include "lib/query/credentialAtomicQuerySigOnChainSmt.circom";
+
+component main{public [userStateInOnChainSmtRoot,
+                        userCorrelationID,
```

Comment:
userCorrelationID should it be public signal?

Reply:
Yes, in initial nullifier implementation the userCorrelationID is a public signal, which is either the verifier id or some random number to keep user session. However, it does not matter as we'll use other approach to calculate nullifiers.

Reply:
Closing this conversation as userCorrelationID not relevant. `UserCorrelationID` input will not exist in the current nullifier schema.